Cython/Compiler/Parsing.py

```diff
@@ -4263,7 +4259,7 @@ def p_value_pattern(s):
         attr_pos = s.position()
         s.next()
         attr = p_ident(s)
-        res = ExprNodes.AttributeNode(attr_pos, obj = res, attribute=attr)
+        res = ExprNodes.AttributeNode(attr_pos, obj = res, attribute = attr)
```

Comment:
Sorry, PEP 8 actually [wants it](https://peps.python.org/pep-0008/#other-recommendations) the other way round.
```suggestion
        res = ExprNodes.AttributeNode(attr_pos, obj=res, attribute=attr)
```
Cython's code base still uses spaces all over the place, for legacy reasons. I'm trying to get rid of those slowly, whenever I modify arguments somewhere.

Reply:
Fair enough. I copied the rest of the file but that clearly wasn't the right thing to do

There's a whole bunch more of these in the new code so I'll go correct them all in a big batch